src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/roles/edit/details/main.js

```diff
@@ -48,8 +48,18 @@ define([
         },
 
         bindDOMEvents: function() {
-            this.sandbox.dom.on('#select-all', 'click', function() {
-                this.sandbox.emit('husky.matrix.set-all');
+            this.sandbox.dom.on('#change-all', 'click', function() {
+                if(!!$('#change-all').hasClass('set-all')) {
```

Comment:
Classes should be used for styling. Keep this knowledge inside the component instead of moving it to the DOM (like `this.saved`).